addons/pkg/util/addon_util.go

```diff
@@ -218,3 +227,97 @@ func IsPackageInstallPresent(ctx context.Context,
 
 	return true, nil
 }
+
+// AddFinalizerToCR adds finalizer to the config CR if not present and
+// returns true if finalizer is added
+func AddFinalizerToCR(
```

Comment:
Question: Do we need finalizers anymore?

Reply:
Not needed. I removed them now.